### PR TITLE
Cosmetics (NFC)

### PR DESCRIPTION
**Pull request by danra**
#### Resolved bug number: ([SR-](https://bugs.swift.org/browse/SR-))

---

Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

 **Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| OS X platform | @swift-ci Please test OS X platform |
| Linux platform | @swift-ci Please test Linux platform |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

- Comments minor fixes
- Align functions' parameters to same column

### CODE REVIEW

File: include/swift/Subsystems.h [hunk 1]
```diff
@@ -232,28 +232,30 @@ namespace swift {
...
 
   /// Turn the given Swift module into either LLVM IR or native code
   /// and return the generated LLVM IR module.
-  std::unique_ptr<llvm::Module> performIRGeneration(IRGenOptions &Opts,
-                                                    SourceFile &SF,
-                                                    SILModule *SILMod,
-                                                    StringRef ModuleName,
-                                                 llvm::LLVMContext &LLVMContext,
-                                                    unsigned StartElem = 0);
+  std::unique_ptr<llvm::Module>
```
Comment (gottesmm) [inline include/swift/Subsystems.h]: Can you see what clang-format does on this text? Try using git-clang-format. Many of us use that as our baseline of changes like this, so matching what clang-format does will reduce future changes by clang-format.

**Comment by gottesmm** [comment]
@swift-ci Please smoke test

File: include/swift/Subsystems.h (hunk 1 above)
Comment (danra) [inline include/swift/Subsystems.h]: Done, fixed according to git-clang-format

**Comment by gottesmm** [comment]
This LGTM.

**Pull request merged**